format/index.md

```diff
@@ -480,9 +480,6 @@ For example:
       "Steve Klabnik",
       "Dan Gebhardt"
     ]
-  },
-  "data": {
-    // ...
```

Comment:
Here, I've removed the `"data"` section, since the `// ...` comment inside is not legal. Maybe it would make more sense to keep it and replace the `...` with `"type"` and `"id"` instead?